components/osc-host-application/ApplicationForm.js

```diff
@@ -192,11 +184,21 @@ const messages = defineMessages({
   },
   aboutYourCommunityTitle: {
     id: 'HostApplication.form.aboutYourCommunity',
-    defaultMessage: 'About your community {optional} {padlock}',
+    defaultMessage: 'About your community {optional}',
```

Comment:
```suggestion
    defaultMessage: 'About your community',
```
still confusing to have this marked like this, maybe lead with text at this point?

Reply:
perhaps help text to read:

> If your community's work is broader than a single open source project please share information about your activities license so we can properly consider your application.

Reply:
Switching references from "events" to "activities" in the Community section could make sense, seems a bit too specific with events (although that might be the most common thing?)

"Links to previous activities" instead of "Links to previous events"?

Reply:
Should we fix this as part of the issue instead? https://github.com/opencollective/opencollective/issues/6053